src/main/scala/firrtl/transforms/ConstantPropagation.scala

```diff
@@ -124,6 +124,28 @@ class ConstantPropagation extends Transform with DependencyAPIMigration {
     case _                      => false
   }
 
+  val options = Seq(
+    new ShellOption[Unit](
+      longOption = "no-constant-propagation",
+      toAnnotationSeq = (_: Unit) => Seq(NoConstantPropagationAnnotation),
```

Comment:
Nit: Does this need the explicit type?

```suggestion
      toAnnotationSeq = _ => Seq(NoConstantPropagationAnnotation),
```